modules/images/webp-uploads/load.php

```diff
@@ -72,6 +72,25 @@ function webp_uploads_create_sources_property( array $metadata, $attachment_id )
 		'crop'   => false,
 	);
 
+	$sizes           = wp_get_registered_image_subsizes();
+	$image_size_name = 'full';
+
+	if ( ! empty( $sizes ) ) {
+		foreach ( $sizes as $image_size_key => $size ) {
+			if (
+				isset( $size['width'] )
+				&& isset( $size['height'] )
+				&& isset( $size['crop'] )
+				&& $size['width'] === $original_size_data['width']
+				&& $size['height'] === $original_size_data['height']
+				&& $size['crop'] === $original_size_data['crop']
+			) {
+				$image_size_name = $image_size_key;
+				break;
+			}
+		}
+	}
+
```

Comment:
This isn't really accurate, but more importantly it isn't needed: This function only deals with the `full` image size (for all other image sizes the `webp_uploads_generate_image_size()` function is used).

So this set of changes can be removed entirely, instead we can pass the hard-coded string `full` to `webp_uploads_generate_additional_image_source()` below.

Reply:
@felixarntz This changes has been done now.